x-pack/plugins/ml/public/application/model_management/model_actions.tsx

```diff
@@ -33,6 +34,80 @@ import { isTestable, isDfaTrainedModel } from './test_models';
 import { ModelItem } from './models_list';
 import { usePermissionCheck } from '../capabilities/check_capabilities';
 
+const ViewTrainingDataAction: FC<{
+  item: ModelItem;
+}> = ({ item }) => {
+  const urlLocator = useMlLocator()!;
+  const {
+    services: {
+      application: { navigateToUrl },
+    },
+  } = useMlKibana();
+
+  const handleClick = async () => {
+    if (item.metadata?.analytics_config === undefined) return;
+
+    const analysisType = getAnalysisType(
+      item.metadata?.analytics_config.analysis
+    ) as DataFrameAnalysisConfigType;
+
+    const url = await urlLocator.getUrl({
+      page: ML_PAGES.DATA_FRAME_ANALYTICS_EXPLORATION,
+      pageState: {
+        jobId: item.metadata?.analytics_config.id as string,
+        analysisType,
+        ...(analysisType === 'classification' || analysisType === 'regression'
+          ? {
+              queryText: `${item.metadata?.analytics_config.dest.results_field}.is_training : true`,
+            }
+          : {}),
+      },
+    });
+
+    await navigateToUrl(url);
+  };
+
+  const buttonContent = (
```

Comment:
The styling of the button has changed from before:

<img width="145" alt="image" src="https://github.com/elastic/kibana/assets/7405507/74bb6419-e9d2-4546-ae8c-7a9350c4fce7">

now:

<img width="218" alt="image" src="https://github.com/elastic/kibana/assets/7405507/5985a3c7-cb7c-4fd6-83c5-dad76aa937f8">

Can the previous style be retained (to show the icon without text, and the same blue font color)?

Reply:
Styling for actions in https://github.com/elastic/kibana/pull/171061/commits/8c7094debba6f09e449d5ab8a8eda6f797b398c5 LGTM